### ISSUE TITLE
Support for authProvider option

### ISSUE THREAD

**Issue by doubleoevan** [masumsoft/express-cassandra#38]
I would like to be able to pass in the authProvider as an option like this...
```
import cassandra from 'express-cassandra';
import driver from 'cassandra-driver';
import config from '../config';

// configure the authProvider
const { contactPoints, cassandraUsername, cassandraPassword, keyspace } = config;
const authProvider = new driver.auth.PlainTextAuthProvider(cassandraUsername, cassandraPassword);

// export the cassandra connection
export default cassandra.createClient({
  clientOptions: {
    contactPoints,
    authProvider,
    keyspace
  }, etc...
});
```
Here is a pull request: https://github.com/masumsoft/express-cassandra/pull/37
Thanks!
Evan

**Comment by datpham23** [comment]
+1

**Comment by masumsoft** [comment]
fixed in v0.8.2
